developer/web-api/users.md

```diff
@@ -333,6 +333,17 @@ curl -d @replica.json "localhost/api/33/users/N3PZBUlN8vq/replica"
   -H "Content-Type:application/json" -u admin:district
 ```
 
+### Reset user password { #webapi_user_reset }
+
+User administrators (with appropriate rights) can reset another user's account
+by triggering password recovery. Once triggered an email is send to the user
```

Comment:
Use "sent" instead of "send".

Reply:
You are right - I had to look this up thinking in combination with _is_ it should be _send_ but it is still _sent_. :+1: